corehq/apps/app_manager/templates/app_manager/app_view.html

```diff
@@ -52,6 +52,7 @@
         rows: 1,
         saveValueName: 'name',
         nodeName: 'input',
+        allow_edit: '{{ request.couch_user.can_edit_apps|yesno:"true,false" }}',
```

Comment:
Expecting "true" or "false" as a string value is confusing, can this accept real booleans instead?

Reply:
Sure, there was some issue in the beginning, but made it a boolean now.